src/Extensions/Controls/index.js

```diff
@@ -100,25 +100,25 @@ Leaflet.Control.GroupedLayers = Leaflet.Control.extend({
 		Leaflet.DomEvent.disableScrollPropagation(container)
 
     var section = this._section = Leaflet.DomUtil.create('div', `${baseClass}__list`)
-
+      
     if (this.options.collapsed) {
-      if (!Leaflet.Browser.android) {
-        Leaflet.DomEvent.on(container, 'mouseover', this._expand, this).on(container, 'mouseout', this._collapse, this)
-      }
       var link = this._layersLink = Leaflet.DomUtil.create('a', baseClass + '__toggle', container)
       link.href = '#'
       link.title = 'Layers'
 
+      var closeLink = this._closeLink = Leaflet.DomUtil.create('a', 'smbc-control-layers__list-close-buttons', section)
```

Comment:
updated classname to use baseClass variable and remove 's' from buttons due to DS changes and will approve.